.github/workflows/regression-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Regression Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Python ${{matrix.python-version}}
+    runs-on: ubuntu-20.04
```

Comment:
We could also run on mac OS and Windows, no? ;-) Maybe a good idea, since there are some windows MacOS users from time to time

Reply:
We can although not sure if needed.

Reply:
let's do it in other PR, if wanted